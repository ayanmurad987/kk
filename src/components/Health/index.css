.healthBar {
    border: 2px solid;
    width: 100%;
    height: 100%;
}
.healthBar > div {
    width: 80%;
    height: 100%;
    border: none;
    transition: width 1s
}