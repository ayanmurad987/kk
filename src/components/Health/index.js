import React from 'react'
import './index.css'

export default function Health(props) {
	const { value, id } = props
	const setColor = () => {
		if (value >= 0 && value <= 50) return '#5cb85c'
		if (value >= 51 && value <= 70) return '#f0ad4e'
		if (value >= 71 && value <= 100) return '#f0ad4e'
  }
  
	return (
		<div className='healthBar'>
			<div id={id} style={{width:`${value}%`, backgroundColor: setColor()}}></div>
		</div>
	);
}
