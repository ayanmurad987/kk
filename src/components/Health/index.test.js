import React from 'react'
import { shallow, mount } from 'enzyme'
import Health from './index'

const mockValue = 25

it('renders Health component without crashing.', () => {
  const wrapper = shallow(<Health {...mockValue} />)
  expect(wrapper).toMatchSnapshot()
})
