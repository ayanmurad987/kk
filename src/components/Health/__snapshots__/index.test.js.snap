// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Health component without crashing. 1`] = `ShallowWrapper {}`;
